src/pages/testingHistory/components/HistoryContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import { CircularProgress, Box } from "@mui/material";
+import { useAppSelector } from "store/store";
+
+import type { Run } from 'components/CreateCertificate/CreateCertificate';
+import AppTable from "./AppTable";
+import { useEffect } from "react";
+
+interface AppHistory {
+    appName: string;
```

Comment:
In almost all files we have 2 spaces tabs, but it looks like this file has 4 spaces tabs.
Can we fix the tabs on this file?